docker/config/l1-node/generate-genesis.sh

```diff
@@ -0,0 +1,39 @@
+# Script to modify config files to enable Prague/Electra to run for the local stack
+# Runs on the 'l1-node-genesis-generator' Docker image entrypoint
+
+original_el_genesis_json_path="/config/genesis.json"
+original_cl_network_config_path="/config/network-config.yml"
+output_dir="/data/l1-node-config"
+modified_el_genesis_json_path=$output_dir/$(basename -- $original_el_genesis_json_path)
+modified_cl_network_config_path=$output_dir/$(basename -- $original_cl_network_config_path)
+
+mkdir -p $output_dir
+cp $original_el_genesis_json_path $modified_el_genesis_json_path
+cp $original_cl_network_config_path $modified_cl_network_config_path
+
+# Early exit if $PRAGUE feature flag is off
+if [ -z "$PRAGUE" ]; then
+  exit 0
+fi
+
+OS=$(uname);
+prague_time=$(    
+    if [ $OS = "Linux" ]; then
+        date -d "+32 seconds" +%s;
+    elif [ $OS = "Darwin" ]; then
+        date -v +32S +%s;
+    fi
+)
+
+# Add Prague config to modified Besu config
+jq --argjson prague_time "$prague_time" '.config.pragueTime = $prague_time' $original_el_genesis_json_path > $modified_el_genesis_json_path
```

Comment:
I did a quick search, seems like jq doesn't have in-place replacement like `sed -i`, and if that's the case, the above looks good